examples/cra-kitchen-sink/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "react-scripts build",
     "build-storybook": "build-storybook -s public",
     "eject": "react-scripts eject",
-    "start": "react-scripts start",
+    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
```

Comment:
the [error message](https://github.com/storybooks/storybook/issues/4376#issue-369379922) recommends using `.env` file for that

Reply:
Sorry, you're right. I thought if it was in a `.env` file we'd have to source it resulting in the same amount of code. Turns out `react-scripts` will do that for us.

However, `.env` is in `examples/cra-kitchen-sink/.gitignore` so it can't be committed. What is the better solution here, leave things as they are, or remove `.env` from the git ignore?

Reply:
Yes, let's remove it from gitignore